src/zproxy.c

```diff
@@ -183,6 +198,32 @@ s_self_selected_socket (zmsg_t *request)
     return socket;
 }
 
+static int
+s_get_int_value (zmsg_t *request)
+{
+    char *value = zmsg_popstr (request);
```

Comment:
strings from zmsg_popstr need to be freed

Reply:
Thank you!